<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FloatParm</class>
 <widget class="QWidget" name="FloatParm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QCheckBox" name="checkBox"/>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="valueWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueWidget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>13</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
